cmd/gcs-sidecar/main.go

```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/Microsoft/hcsshim/internal/gcs/prot"
 	shimlog "github.com/Microsoft/hcsshim/internal/log"
 	"github.com/Microsoft/hcsshim/internal/oc"
-	"github.com/Microsoft/hcsshim/internal/pspdriver"
```

Comment:
is there a reason this was moved to securitypolicy package?

Reply:
All host validation code is moved to `securitypolicy` package. This code is now used only there.